elastic/integrations
# Fix ECS source.ip field in users

## What does this PR do?

 This fixes a straggling `keyword` mapping of `source.ip` in the users datastream, reported here: https://github.com/elastic/kibana/issues/73797

## Checklist


## How to test this PR locally

- Pull down PR, build and run elastic-stack with agent on linux, and `linux/users` enabled.
- You should be able to reproduce this on a linux server with remote users logged in.
- Make sure there's no mapping conflicts.
